src/Trace/Propagator/TraceContextFormatter.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright 2017 OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCensus\Trace\Propagator;
+
+use OpenCensus\Trace\TraceContext;
+
+/**
+ * This format using a human readable string encoding to propagate TraceContext.
+ * See https://github.com/TraceContext/tracecontext-spec/blob/master/trace_context/HTTP_HEADER_FORMAT.md
+ * for the definition.
+ */
+class TraceContextFormatter implements FormatterInterface
+{
+    const CONTEXT_HEADER_FORMAT = '/([0-9a-f]{2})-(.*)/';
```

Comment:
Is it possible that the hex values uses uppercase letters?

Reply:
It's not excluded in the spec, so I'll make sure it works for both.